### DISCUSSION TITLE
Getting error while adding  const workbook = new ExcelJS.Workbook(); in next js

### DISCUSSION THREAD

**Discussion by Shahekta51995** [exceljs/exceljs#2550] [General]
I am getting error while adding new ExcelJS.Workbook(); 

package:
    "exceljs": "^4.3.0",
    "next": "^13.4.12",

import ExcelJS from "exceljs"
 
const downloadxls = async (data: any) => {
    try {
  
      const workbook = new ExcelJS.Workbook();
       const worksheet = workbook.addWorksheet('Sheet1');
       worksheet.addRow(['Name', 'Age']);
       worksheet.addRow(['John Doe', 30]);
  
      const buffer = await workbook.xlsx.writeBuffer();
      const blob = new Blob([buffer], { type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' });
      const url = window.URL.createObjectURL(blob);
      const a = document.createElement('a');
      a.href = url;
      a.download = 'example.xlsx';
       a.click();
       window.URL.revokeObjectURL(url);
      
    } catch (err) {
      console.log("errrrrrrrr", err);
    }
  };

**Comment by juanbercoff** [comment]
Same issue
